api/swagger-spec/swagger.yml

```diff
@@ -275,6 +301,13 @@ definitions:
         type: array
         items:
           $ref: "#/definitions/componentVersion"
+  doctorInfo:
+    type: object
+    required:
+      - cluster
+    properties:
+      cluster:
+        $ref: "#/definitions/cluster"
```

Comment:
I think we should create a new model here. Maybe call it `doctorData`. It would look something like this:

```
{
    id: {uuid}
    cluster_id: {cluster id}
    components: {componentVersion}
    data: {generic object}
}
```

The `generic object` data property above would evolve over time to contain the payload of the deis doctor report; the `components` property would contain the basic deis cluster version metadata.

@slack might want to sanity check this proposal!